.github/workflows/ci.yml

```diff
@@ -70,27 +71,27 @@ jobs:
               ;;
           esac
 
-      - name: Configure
+      - name: Configure (${{ matrix.build-type }})
         run: |
           case ${{ runner.os }} in
             Windows*)
               cmake -S . -B cmake-build \
-                -DCMAKE_BUILD_TYPE=${{ matrix.configuration }} \
+                -DCMAKE_BUILD_TYPE=${{ matrix.build-type }} \
                 -G "Visual Studio 16 2019" \
                 -A x64
               ;;
             *)
               cmake -S . -B cmake-build \
-                -DCMAKE_BUILD_TYPE=${{ matrix.configuration }}
+                -DCMAKE_BUILD_TYPE=${{ matrix.build-type }}
               ;;
           esac
 
       - name: Build
-        run: cmake --build cmake-build
+        run: cmake --build cmake-build --config ${{ matrix.build-type }}
```

Comment:
Is this needed because both release and debug builds are in the same directory? Then perhaps it is better to pass a different build directory like `cmake -S . -B cmake-build-{{ matrix.build-type }}` to avoid weird bugs due to muti-configuration builds?

Reply:
I think this is Visual Studio specific stuff. For more see https://stackoverflow.com/questions/19024259/how-to-change-the-build-type-to-release-mode-in-cmake

Reply:
Ah, ok. Good to know.